<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="waioz-Lenovo-B460e" failures="2" tests="8" name="Test" time="9.787" errors="0" timestamp="2022-12-28T15:55:35 IST">
  <testcase classname="com.RestAssured.TestCases.TC003_Post_Employee_Record" name="@BeforeClass createEmployee" time="1.223">
    <failure type="io.restassured.path.json.exception.JsonPathException" message="Failed to parse the JSON document">
      <![CDATA[io.restassured.path.json.exception.JsonPathException: Failed to parse the JSON document
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1002)
at io.restassured.path.json.JsonPath$4.doParseWith(JsonPath.java:967)
at io.restassured.path.json.JsonPath$JsonParser.parseWith(JsonPath.java:1047)
at io.restassured.path.json.JsonPath.get(JsonPath.java:202)
at com.RestAssured.TestCases.TC003_Post_Employee_Record.createEmployee(TC003_Post_Employee_Record.java:61)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: groovy.json.JsonException: Lexing failed on line: 1, column: 1, while reading '<', no possible valid JSON value or punctuation could be recognized.
at groovy.json.JsonLexer.nextToken(JsonLexer.java:86)
at groovy.json.JsonLexer$nextToken.call(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:130)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parse(ConfigurableJsonSlurper.groovy:97)
at io.restassured.internal.path.json.ConfigurableJsonSlurper$parse.callCurrent(Unknown Source)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:171)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.path.json.ConfigurableJsonSlurper.parseText(ConfigurableJsonSlurper.groovy:83)
at io.restassured.path.json.JsonPath$4$1.method(JsonPath.java:965)
at io.restassured.path.json.JsonPath$ExceptionCatcher.invoke(JsonPath.java:1000)
... 30 more
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass createEmployee -->
  <testcase classname="com.RestAssured.TestCases.TC002_GET_SingleEmployeeRecord" name="checkStatusLine" time="0.046">
    <failure type="java.lang.AssertionError" message="HTTP/1.1 429 Too Many Requests expected:&amp;lt;HTTP/1.1 200 OK&amp;gt; but was:&amp;lt;Status line is HTTP/1.1 200 OK&amp;gt;">
      <![CDATA[java.lang.AssertionError: HTTP/1.1 429 Too Many Requests expected:<HTTP/1.1 200 OK> but was:<Status line is HTTP/1.1 200 OK>
at com.RestAssured.TestCases.TC002_GET_SingleEmployeeRecord.checkStatusLine(TC002_GET_SingleEmployeeRecord.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- checkStatusLine -->
  <testcase classname="com.RestAssured.TestCases.TC002_GET_SingleEmployeeRecord" name="checkStausCode" time="0.029">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;429&amp;gt; but was:&amp;lt;200&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<429> but was:<200>
at com.RestAssured.TestCases.TC002_GET_SingleEmployeeRecord.checkStausCode(TC002_GET_SingleEmployeeRecord.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- checkStausCode -->
  <testcase classname="com.RestAssured.TestCases.TC001_GET_AllEmployees" name="checkResponse" time="0.096"/>
  <testcase classname="com.RestAssured.TestCases.TC001_GET_AllEmployees" name="checkStatusCode" time="0.008"/>
  <testcase classname="com.RestAssured.TestCases.TC001_GET_AllEmployees" name="checkStatusLine" time="0.007"/>
  <testcase classname="com.RestAssured.TestCases.TC003_Post_Employee_Record" name="checkResponsePayload" time="0.0">
    <skipped/>
  </testcase> <!-- checkResponsePayload -->
  <testcase classname="com.RestAssured.TestCases.TC003_Post_Employee_Record" name="checkStatusLine" time="0.0">
    <skipped/>
  </testcase> <!-- checkStatusLine -->
  <testcase classname="com.RestAssured.TestCases.TC003_Post_Employee_Record" name="checkStausCode" time="0.0">
    <skipped/>
  </testcase> <!-- checkStausCode -->
</testsuite> <!-- Test -->
